verbose_debugging_reports.md

```diff
@@ -256,6 +256,13 @@ had [insufficient budget][].
 
 Additional fields: `limit`
 
+#### `trigger-aggregate-incompatible-max-reports`
```

Comment:
This should be added to `triggerAggregatableDebugTypes` as well.

Reply:
Added to `ts/src/constants.ts`